src/sbvr-api/sbvr-utils.ts

```diff
@@ -289,8 +289,6 @@ const prettifyConstraintError = (
 				'Data is referenced by ' + sqlNameToODataName(matches[1]) + '.',
 			);
 		}
-
-		throw err;
```

Comment:
did you mean to drop this `throw` ?

Reply:
Yes in order to avoid leaking info in the case it's a new constraint error that hasn't got special handling, but I've changed it to instead override the error which would be better than having it go to the default `Database error` override message